components/serverless/config/crd/bases/serverless.kyma-project.io_functions.yaml

```diff
@@ -408,6 +408,10 @@ spec:
                     - source
                     type: object
                 type: object
+                x-kubernetes-validations:
+                - message: Use GitRepository or Inline source
```

Comment:
IMO this `message` say something else than `rule`. Maybe we can have `message` similar to "Use GitRepository or Inline source, not both". It's problem in difference between natural language and logic language.

Reply:
In `Resource configuration` we have something like that: `Use profile or resources` I can lowercase the `Inline` and `GitRepository`

Reply:
ok